frontend/src/containers/allocations.js

```diff
@@ -33,7 +49,7 @@ class Allocations extends Component {
                                                 <td><NomadLink allocId={allocation.ID} short="true"/></td>
                                                 <td><NomadLink jobId={allocation.JobID} short="true"/></td>
                                                 <td>{allocation.ClientStatus}</td>
-                                                <td>{allocation.DesiredStatus}</td>
+                                                <td>{this.renderDesiredStatus(allocation)}</td>
```

Comment:
This is really nice! I wonder if we should make it clear in the UI that more information is to be had by hovering over the item. What do you think?